bin/templates/cordova/lib/builders/ProjectBuilder.js

```diff
@@ -300,27 +301,19 @@ class ProjectBuilder {
 
 module.exports = ProjectBuilder;
 
-function apkSorter (fileA, fileB) {
-    const archSpecificRE = /-x86|-arm/;
+const apkSorter = compareFunc([
+    // Sort arch specific builds after generic ones
+    apkPath => /-x86|-arm/.test(apkPath),
 
-    const unsignedRE = /-unsigned/;
+    // Sort unsigned builds after signed ones
+    apkPath => /-unsigned/.test(apkPath),
 
-    // De-prioritize arch-specific builds & unsigned builds
-    const lower = (fileName) => {
-        return archSpecificRE.exec(fileName)
-            ? -2
-            : unsignedRE.exec(fileName)
-                ? -1
-                : 0;
-    };
+    // Sort by file modification time, latest first
+    apkPath => -fs.statSync(apkPath).mtime.getTime(),
```

Comment:
Is the `-` in `-fs.statSync(apkPath).mtime.getTime()` correct?

Reply:
if it's not, would be "funny" that the tests pass anyway...

Reply:
Yes, the minus creates a descending sort by `mtime`, which means bigger/later values will come before smaller/earlier ones.